kokoro/linux/build.sh

```diff
@@ -40,6 +40,15 @@ export ANDROID_NDK_HOME=$PWD/android-ndk-r21
 # Get recent build tools
 echo y | $ANDROID_HOME/tools/bin/sdkmanager --install 'build-tools;29.0.2'
 
+# Get the JDK from our mirror. This needs to be after the Android update above.
```

Comment:
I would say the "why": `This needs to be after the Android update above which requires Java 8`.